src/API/Site/Controllers/MerchantCenter/PriceBenchmarksController.php

```diff
@@ -224,26 +279,79 @@ protected function get_schema_properties(): array {
 					'title'     => [ 'type' => 'string' ],
 				],
 			],
-			'effectiveness'   => [
+			'effectiveness'                => [
 				'description' => __( 'Effectiveness score.', 'google-listings-and-ads' ),
 				'type'        => 'number',
 			],
-			'regular_price'   => [
+			'regular_price'                => [
 				'description' => __( 'Regular price of the product.', 'google-listings-and-ads' ),
 				'type'        => 'number',
 			],
-			'price_on_google' => [
+			'price_on_google'              => [
 				'description' => __( 'Price of the product on Google.', 'google-listings-and-ads' ),
 				'type'        => 'number',
 			],
-			'price_gap'       => [
+			'price_gap'                    => [
 				'description' => __( 'Price gap between the regular price and the price on Google.', 'google-listings-and-ads' ),
 				'type'        => 'number',
 			],
-			'suggested_price' => [
+			'suggested_price'              => [
 				'description' => __( 'Suggested price for the product.', 'google-listings-and-ads' ),
 				'type'        => 'number',
 			],
+			'clicks'                       => [
+				'description' => __( 'Current clicks.', 'google-listings-and-ads' ),
+				'type'        => [ 'integer', 'null' ],
+			],
+			'conversions'                  => [
+				'description' => __( 'Current conversions.', 'google-listings-and-ads' ),
+				'type'        => [ 'integer', 'null' ],
+			],
+			'predicted_clicks_change'      => [
+				'description' => __( 'Expected uplift in clicks (fraction).', 'google-listings-and-ads' ),
+				'type'        => [ 'string', 'null' ],
+			],
+			'predicted_conversions_change' => [
+				'description' => __( 'Expected uplift in conversions (fraction).', 'google-listings-and-ads' ),
+				'type'        => [ 'string', 'null' ],
+			],
+		];
+	}
+
+	/**
+	 * Retrieves the parameters for price benchmarks.
+	 *
+	 * @return array An associative array of parameters for price benchmarks.
+	 */
+	public function get_price_benchmarks_params(): array {
+		return [
+			'ids'      => [
```

Comment:
This defines `ids` as the arg that is supported, but the logic seems to use the `id` param instead. These should align.

It's ok to support `fields` (as long as it's working)  but I don't think we need to support `interval` for now

Reply:
Addressed in https://github.com/woocommerce/google-listings-and-ads/pull/2891/commits/a0477e04ba2a690c51932542c2478157f637d412